### DISCUSSION TITLE
Error Message: "Expected Lenght for scale calculation."

### DISCUSSION THREAD

**Discussion by KadeKatrak** [pubpub-zz/ppInk#307] [General]
Thank you for making this!

I'm not very tech savvy, but was able to download and use ppink with my Huion drawing tablet. It is working well so far with one small exception. I regularly get an error message that says "expected lenght for scale calculation (Pixel)". I either hit "Ok" or "cancel" and the error message goes away and it continues working. Is there any way I can prevent it from popping up? And if that is not possible, can I correct the spelling error? 

I am hoping to use ppinc to help tutor students remotely and would rather not have the popup interrupt. But if it has to, I would like it to be spelled correctly.
![image](https://github.com/user-attachments/assets/87fda1e6-913b-4c63-a90b-82ae57a446f0)

Edit: Never mind. I found the answer in an earlier question. I disabled the length calculator by right-clicking the PPink logo -> Options --> Measurement Tools --> and unchecking the box.

Thank you again!

**Comment by pubpub-zz** [comment]
Happy you enjoy. Do not hesitate to star the project.
The pop up is not an error but a way to define the scale factor. this pop up occurs because you are drawing lines or hand with the right click (maybe because you are pressing too long the arrow). If you don't need this uncheck the "measure while drawing" in the options:
![image](https://github.com/user-attachments/assets/e56c3664-372d-4c35-a7ba-4fa944ec0701)

**Reply by KadeKatrak** [reply]
Thanks. I have now starred the project! I didn't even know that was a thing. As I mentioned, I'm new to this. And thanks again. It is a very helpful tool.
